homeassistant/components/raspihats.py

```diff
@@ -12,7 +12,8 @@
     EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP
 )
 
-REQUIREMENTS = ['raspihats==2.2.1']
+REQUIREMENTS = ['raspihats==2.2.3'
```

Comment:
Missing comma. This one just concats the strings.

    >>> REQUIREMENTS = ['raspihats==2.2.3'
    ...                 'smbus-cffi==0.5.1']
    >>> REQUIREMENTS
    ['raspihats==2.2.3smbus-cffi==0.5.1']
    >>>

Reply:
ah, ty! :)